docs/gallery.py

```diff
@@ -114,7 +126,7 @@ def pcoords():
 
 def pca():
     X, y = load_credit()
-    colors = np.array(['r' if yi else 'b' for yi in y])
+    colors = np.array(["r" if yi else "b" for yi in y])
     oz = PCADecomposition(scale=True, color=colors, proj_dim=3)
```

Comment:
This is super annoying and I'm sorry about it - but the tests are complaining because this needs to be `PCA`. This is my fault for suggesting we change it!

Reply:
Doh! Fixed now 🙂